app/src/main/java/de/grobox/transportr/trips/search/DirectionsFragment.kt

```diff
@@ -111,8 +111,10 @@ class DirectionsFragment : TransportrFragment() {
             fromLocation.setLocation(it)
             if (it != null) toLocation.requestFocus()
         })
+        viewModel.fromLocation.observe(viewLifecycleOwner, { onLocationsChanged() })
         viewModel.viaLocation.observe(viewLifecycleOwner, { viaLocation.setLocation(it) })
         viewModel.toLocation.observe(viewLifecycleOwner, { toLocation.setLocation(it) })
+        viewModel.toLocation.observe(viewLifecycleOwner, { onLocationsChanged() })
```

Comment:
```suggestion
        viewModel.toLocation.observe(viewLifecycleOwner, {
            toLocation.setLocation(it)
            onLocationsChanged()
        })
```

Same suggestion for above.